qemu/tests/cfg/tpm_with_tss.cfg

```diff
@@ -0,0 +1,25 @@
+- tpm_with_tss:
+    virt_test_type = qemu
+    type = tpm_with_tss
+    only Linux
+    tpms = tpm0
+    tpm_version_tpm0 = 2.0
+    tpm_type_tpm0 = emulator
+    x86_64:
+        only q35
+        only ovmf
+        required_qemu= [4.2.0,)
+        tpm_model_tpm0 = tpm-crb
+    ppc64le, ppc64:
+        required_qemu= [5.0.0,)
+        tpm_model_tpm0 = tpm-spapr
+    aarch64:
+        required_qemu = [5.1.0,)
+        tpm_model_tpm0 = tpm-tis-device
+    tpm_device = /dev/tpmrm0
```

Comment:
Why check /dev/tpmrm0 instead of /dev/tmp0 ? Is it required by the test suite?

Reply:
This test case only tests TPM 2.0, and here is the difference between TPM 1.2 and TPM 2.0.
> There are two very different TPM specifications: 1.2 and 2.0, which also use different software stacks.
>
> - TPM 1.2 uses the "TrouSerS" TSS (TCG software stack) by IBM, which is packaged as [trousers](https://aur.archlinux.org/packages/trousers/)AUR (tcsd) and [tpm-tools](https://aur.archlinux.org/packages/tpm-tools/)AUR (userspace). All software access the TPM through the tcsd daemon.
> - TPM 2.0 allows direct access via /dev/tpm0 (one client at a time), managed access through the [tpm2-abrmd](https://archlinux.org/packages/?name=tpm2-abrmd) resource manager daemon, or kernel-managed access via /dev/tpmrm0. There are two choices of userspace tools, [tpm2-tools](https://archlinux.org/packages/?name=tpm2-tools) by Intel and [ibm-tss](https://aur.archlinux.org/packages/ibm-tss/)AUR by IBM.
>
> TPM 2.0 requires [UEFI](https://wiki.archlinux.org/title/UEFI) boot; BIOS or Legacy boot systems can only use TPM 1.2.
> Some TPM chips can be switched between 1.2 and 2.0 through a firmware upgrade (which can be done only a limited number of times).

So seems `tpmrm0` is more suitable.

Reply:
From the device test point of view I prefer /dev/tpm0. From the definition of tpmrm /dev/tpmrm0 (kernel resident resource manager), I am not sure if these two have same functionality. 

But from tss2-tcti tests,  /dev/tpmrm0 is configured for the tests. I am ok with /dev/tpmrm0 here. 

Polarion test step shows:
cd tpm2-tss && ./bootstrap && ./configure --enable-unit --with-device=/dev/tpm0

Please make them identical to avoid confusing in the future.

Reply:
OK, will update the manual test, thanks.